### ISSUE TITLE
Check citation scheme and refsDecl for /tlg1205/tlg002/tlg1205.tlg002.perseus-grc2.xml

### ISSUE THREAD

**Issue by sonofmun** [OpenGreekAndLatin/First1KGreek#121]

**Comment by jduff-chs** [comment]
Same structure issues as #119, milestones instead of second level reference. I'll add sections and move the <p>s inside of them.

**Comment by jduff-chs** [comment]
I'm in the process of fixing this, but it's causing a strange problem in Nemo, where the sectioning becomes out of order, and two whole chapters, 4 and 5, are not recognized at all. The order becomes Chapter 1-3, Chapter 20-24, Chapter 6-19, then Chapter 25. Yet, HookTest recognizes all 25 chapters and 126 sections just fine, and passes. I've troubleshooted it a few times, even starting from scratch, but the problem in Nemo persists. Has anyone ever encountered an issue in Nemo like this?

@annettegessner @sonofmun ?

**Comment by annettegessner** [comment]
I have never encountered something like this. I fear I have to direct you with Nemo questions to poor overworked @PonteIneptique again, sorry Thibault!

**Comment by PonteIneptique** [comment]
Could you upload your test results ? Because these are the most important informations (first) and Nemo only comes as a checking tool :)

**Comment by annettegessner** [comment]
@jduff-chs: I don't think you've made a pull request yet. Please do so, even if the XML doesn't work, so Thibault can have a look at it. Thanks!

**Comment by jduff-chs** [comment]
I've made a pull request, where you can see the edits I've made, and the .zip file below contains the file in question, as well as the HookTest output, which I will also paste here:

```
Run at 2016-06-15 17:53:12
Testing /tlg1205/tlg002/tlg1205.tlg002.perseus-grc2.xml
File parsing passed
Parsed
File ingesting in MyCapytain passed
URN informations passed
urn:cts:greekLit:tlg1205.tlg002.perseus-grc2
Naming conventions passed
RefsDecl parsing passed
2 citation's level found
Passage level parsing passed
25 found
126 found
Unique nodes found by XPath passed
Available in inventory passed
Epidoc DTD validation passed
```

[issue-121.zip](https://github.com/OpenGreekAndLatin/First1KGreek/files/316804/issue-121.zip)

**Comment by jduff-chs** [comment]
And here's the html from the Nemo page, you can see that things are oddly being placed out of order.


[Nemo Output re issue-121.txt](https://github.com/OpenGreekAndLatin/First1KGreek/files/316821/Nemo.Output.re.issue-121.txt)

**Comment by PonteIneptique** [comment]
So. few things : the error seems to come from Nemo, I'll need to investigate it. It's not an issue from the text. Note that the most important to work is the API request : `/api/cts?request=GetValidReff&urn=[your urn]&leevel=[your maximum level]` If all shows well there, then no worries about Nemo.

Another thing : I actually think the second level should not be one, because the other edition we have ( https://github.com/OpenGreekAndLatin/First1KGreek/blob/master/data/tlg1205/tlg002/tlg1205.tlg002.perseus-grc1.xml ) has only one level. I think we could keep the markup but remove the second `refsDecl`. Generally, when you have continuous numbering crossing it means we have two concurrent citation scheme in the same edition (hence 5.24 6.25 etc.)

**Comment by jduff-chs** [comment]
If I removed the second refsDecl, Nemo seems to process everything just fine! 

I will remove that refsDecl from this edition of tlg105/tlg001 and 002 and commit those changes as well in light of that. You're absolutely right in that it seems we don't need *both* sets of concurrent citations. Thanks!

**Comment by jduff-chs** [comment]
Addressed in #267 and #262
